asteroid/utils/hub_utils.py

```diff
@@ -57,7 +46,9 @@ def cached_download(filename_or_url):
     if os.path.isfile(filename_or_url):
         return filename_or_url
 
-    if urlparse(filename_or_url).scheme in ("http", "https"):
+    if filename_or_url.startswith(("http://", "https://")) and not filename_or_url.startswith(
+        huggingface_hub.file_download.HUGGINGFACE_CO_REPO_URL_BASE
```

Comment:
I would just add a `https://huggingface.co/` constant in this local file as this is unlikely to ever change:)

Reply:
(and I would add the prefix-stripping here, unless you feel strongly about it!)

Reply:
No strong feelings, we can do it here.